app/controllers/questions_controller.rb

```diff
@@ -16,9 +22,35 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @question.update_attributes(question_params)
```

Comment:
please use `update` instead of `update_attributes ` if you use `Rails 5`

Reply:
fixed